.dev/gen_roxygen_tests.R

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/Rscript
+# execute from package TLD: ./.dev/gen_roxygen_tests.R
+all_examples <- list.files("tests/testthat/roxygen-examples-complete", full.names = TRUE)
+unique_tests <- unique(sub("-.*", "", basename(all_examples)))
+
+for (test in unique_tests) {
+  out_file <- file.path("tests", "testthat", paste0("test-roxygen-examples-complete-", test, ".R"))
+  extras <- if (test == "15") ', scope = "spaces"' else ""
```

Comment:
Can we avoid magic string here? What's special about `"15"`?

Reply:
PTAL